Source/Android/jni/AndroidCommon/IDCache.cpp

```diff
@@ -271,10 +271,6 @@ jmethodID GetBooleanSupplierGet()
 
 }  // namespace IDCache
 
-#ifdef __cplusplus
-extern "C" {
```

Comment:
Are you sure everything still works fine even with C++ symbol mangling? I'd expect the `extern "C"` to still be necessary...

Reply:
Ah, oops, I was supposed to always include that and not never :(